homeassistant/components/emoncms/sensor.py

```diff
@@ -158,7 +158,7 @@ def __init__(
             id_for_name = "" if str(sensorid) == "1" else sensorid
             # Use the feed name assigned in EmonCMS or fall back to the feed ID
             feed_name = elem.get("name") or f"Feed {elem['id']}"
-            self._attr_name = f"EmonCMS{id_for_name} {feed_name}"
+            self._attr_name = f"EmonCMS{id_for_name} {elem['tag']} {feed_name}"
```

Comment:
This integration currently does not yet have unique identifiers. A change like this, will thus cause new entity IDs and be a breaking change.

We should first migrate this integration to have a configuration flow, which in return will allow entities to have a unique identifier using the config entry ID.

../Frenck